onnxruntime/test/python/transformers/test_parity_gelu.py

```diff
@@ -223,8 +232,13 @@ def test_cuda(self):
         else:
             gpu = torch.device("cuda")
             for i in self.formula_to_test:
-                self.run_one(self.optimized, gpu, hidden_size=self.hidden_size, formula=i)
+                self.run_one(self.optimized, gpu, hidden_size=self.hidden_size, formula=i, verbose=self.verbose)
 
 
 if __name__ == "__main__":
-    unittest.main()
+    args, remaining_args = parse_arguments(namespace_filter=unittest)
+
+    TestGeluParity.verbose = args.log_verbose
+    TestGeluParity.optimized = args.no_optimize
```

Comment:
The logic seems not correct. Shall it be optimized = not no_optimize?

Reply:
Changed this so no_optimize comes out as optimize. This was just the var output of --no_optimize from argparse